gfx/webrender/src/bindings.rs

```diff
@@ -286,7 +286,7 @@ pub extern fn wr_create(width: u32, height: u32, counter: u32) -> *mut WrState {
     // let notifier = Box::new(Notifier::new(window.create_window_proxy()));
     // renderer.set_render_notifier(notifier);
 
-    let pipeline_id = PipelineId(0, counter);
+    let pipeline_id = PipelineId((layers_id >> 32) as u32, layers_id as u32);
```

Comment:
I'm not sure what you mean by this. We have to do the conversion somewhere, whether it's in C++ or Rust code. And if it's in C++ then we have to create a C++ struct that maps to a PipelineId along with a function to do the conversion which seems like strictly more work.